indico/modules/events/contributions/util.py

```diff
@@ -244,6 +244,15 @@ def get_contributions_for_person(event, person, only_speakers=False):
             .all())
 
 
+def get_contributions_for_user(event, user):
```

Comment:
I guess it's an uncommon cases, but shouldn't we still include contributions where the user is only a submitter w/o having a person link as well?